test/test-utils.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { render as rtlRender } from '@testing-library/react';
+import { configureStore } from '@reduxjs/toolkit';
+import { Provider } from 'react-redux';
+// Import your own reducer
+import * as AllTripReqs from '../src/redux/features/tripReq.feature';
+import pageReducer from '../src/redux/views/pages';
+
+// const { registerReducer } = Auth.default;
+const { tripReqReducer } = AllTripReqs;
+
+function render(
+  ui,
+  {
+    preloadedState,
+    store = configureStore({
+      reducer: { tripRequests: tripReqReducer, page: pageReducer },
+      preloadedState,
+    }),
+    ...renderOptions
+  } = {}
+) {
+  function Wrapper({ children }) {
+    return <Provider store={store}>{children}</Provider>;
+  }
+  return rtlRender(ui, { wrapper: Wrapper, ...renderOptions });
+}
+
+// re-export everything
+export * from '@testing-library/react';
+// override render method
+export { render };
```

Comment:
This whole file is unnecessary since jest using the `jest.setup.js` to bind the redux to components with all the reducers. No need to import the individual reducers just use the render from `jest.setup.js`.